generated-platform-project/quarkus-camel/bom/pom.xml

```diff
@@ -578,7 +578,7 @@
       <dependency>
         <groupId>com.google.api.grpc</groupId>
         <artifactId>proto-google-common-protos</artifactId>
-        <version>2.8.3</version>
+        <version>2.0.1</version>
```

Comment:
And just to be clear: it's part of the second commit that upgrades to 2.10.1.Final.

Reply:
It's because it's in the quarkus-bom, see https://repo1.maven.org/maven2/io/quarkus/quarkus-bom/2.10.1.Final/quarkus-bom-2.10.1.Final.pom

Reply:
@loicmathieu is there a reason why we are using such an old version in the Quarkus BOM?

Reply:
Probably for the gRPC extension.

Reply:
But not sure about the reason for the version, sorry.

Reply:
@gsmet gPRC is for @cescoffier and this is the reason why I think it would be great to use the Google library BOM to avoid dependency divergence issue between the Quarkus BOM and Google Cloud Services.

Because here, I'm afraid it may broke the Google Cloud Services extensions.

Reply:
The version is here: https://github.com/quarkusio/quarkus/blob/main/pom.xml#L76

@michalszynkiewicz @cescoffier why are we enforcing such an old version? It seems to be new? Or at least the platform used a newer version until now (it might be due to a change in how the platform is generated).

Reply:
The version of `proto-google-common-protos` managed by Quarkus can certainly be updated. We are just defining it because we have many users of the "Empty.proto" file.

Reply:
OK, let me add it to Dependabot and see how it goes.

Reply:
> It seems to be new? Or at least the platform used a newer version until now (it might be due to a change in how the platform is generated).

No, I think it's because it was added to the quarkus BOM.

Reply:
I will proceed with the release. I pushed a PR for a Dependabot upgrade. If things are fine, we will push the upgrade to 2.10.2.Final.